amundsen_application/static/js/components/TableDetail/OpenRequestMetadata/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+import { ToggleRequestAction } from 'ducks/notification/types';
+import { toggleRequest } from 'ducks/notification/reducer';
+import { bindActionCreators } from 'redux';
+
+interface StateFromProps {
+  requestIsOpen: boolean;
+}
+
+export interface DispatchFromProps {
+  toggleRequest: () => ToggleRequestAction;
+}
+
+export type OpenRequestMetadataProps = StateFromProps & DispatchFromProps;
+
+interface OpenRequestMetadataState {}
+
+export class OpenRequestMetadata extends React.Component<OpenRequestMetadataProps, OpenRequestMetadataState> {
+  public static defaultProps: Partial<OpenRequestMetadataProps> = {};
+
+  constructor(props) {
+    super(props);
+  }
+
+  openRequest = () => {
+    if (!this.props.requestIsOpen) {
```

Comment:
How does the component close? It shouldn't require someone to actually fill it out and hit send in order to close.

Reply:
there's an X button to close